datacatalog/cloud-client/lookup_entry_test.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+
+# Copyright 2019 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from gcp_devrel.testing import eventually_consistent
+
+import lookup_entry
+
+PROJECT = os.environ['GCLOUD_PROJECT']
+BIGQUERY_DATASET = os.environ['GCLOUD_BIGQUERY_DATASET']
+BIGQUERY_TABLE = os.environ['GCLOUD_BIGQUERY_TABLE']
+PUBSUB_TOPIC = os.environ['GCLOUD_PUBSUB_TOPIC']
+
+
+def test_lookup_bigquery_dataset():
+    @eventually_consistent.call
```

Comment:
Since we aren't creating resources in these tests, what about it is eventually consistent / flakey that it needs this decorator?

Reply:
I thought they were included as a standard, to make sure test will be retried in case of flakiness. But I'll remove them.